server.js

```diff
@@ -2,24 +2,46 @@ const bodyParser = require('body-parser');
 const express = require('express');
 const mongoose = require('mongoose');
 
+const Bear = require('./models.js');
+
 const STATUS_USER_ERROR = 422;
 const STATUS_SERVER_ERROR = 500;
 const server = express();
 
 // allow server to parse JSON bodies from POST/PUT/DELETE requests
 server.use(bodyParser.json());
 
-
-
-
-
-
 // TODO: write your server code here
+server.post('/bears', (req, res) => {
+  const { species, latinName } = req.body;
+  if (!species || !latinName) {
+    res.status(STATUS_USER_ERROR).send({ error: 'Please enter the required information to save a Bear.' });
```

Comment:
Very good error handling. This is still awesome practice even if you end up validating via mongoose

Reply:
I would make sure you return in your if block. otherwise, is something happens on the `send` node will just hang up.